packages/db/postgres/tables.sql

```diff
@@ -271,6 +271,8 @@ CREATE INDEX IF NOT EXISTS upload_user_id_deleted_at_idx ON upload (user_id) WHE
 CREATE INDEX IF NOT EXISTS upload_content_cid_idx ON upload (content_cid);
 CREATE INDEX IF NOT EXISTS upload_updated_at_idx ON upload (updated_at);
 CREATE INDEX IF NOT EXISTS upload_user_id_idx ON upload (user_id);
+CREATE INDEX IF NOT EXISTS upload_name_idx ON upload (name);
+CREATE INDEX IF NOT EXISTS upload_inserted_at_idx ON upload (inserted_at);
```

Comment:
Are these only added for the purposes of sorting after already filtering by `user_id`? If so, I have a suspicion that Postgres might not actually use them, because by the time it's reduced the result set down to the rows that match the `user_id` it _might_ deem that it's then just as quick to sort the actual rows as it is to fetch the relevant index entries and cross reference them against the result rows.

I'm not sure without actually trying it out on some real data. It's just a thought.

Reply:
👍 yep! Would be great to test this with real data.
Is there anyway to export data to staging to test this before going live?

Reply:
I'm not sure. I suspect that for a few rows it might just sort the rows, but when it gets to 1000s of rows it might switch to using the index.

For optimal speed for each possible sorting (name asc, name, desc, date asc, date, desc) I think it would need composite indexes for each of:
* `user_id`, `name` ASC
* `user_id`, `name` DESC
* `user_id`, `inserted_at`, ASC
* `user_id`, `inserted_at`, DESC

But that's getting to be quite a few indexes.

Presumably, even if a user has several thousand uploads, sorting them in memory is probably fairly quick (compared to finding the uploads from the millions in the table). So I'm kind of leaning towards thinking that so long as we've got a `user_id` index, the other indexes are unlikely to make much difference! But I might be wrong.

Reply:
This makes sense to me! Reverted the index changes for now. 
Thanks